docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,5 +7,11 @@ Add a summary of the change and which issue is fixed, including motivation, cont
 - [ ] Code follows the style guidelines
 - [ ] Code commented in relevant places
 - [ ] Changes to necessary documentation made
-- [ ] New and existing unit tests pass locally with changes
-- [ ] Basic manual testing performed after deploying app
+
+# Tests Performed:
+
+- [ ] Upload data to Datastore
+- [ ] Retreive data from Datastore
+- [ ] Map Visualization 
+- [ ] Network Map Visualization 
+- [ ] Basic manual testing performed
```

Comment:
Rename the last bullet as "List other manual test cases performed"

Reply:
It's been added